scale/docs/rest/v6/system.rst

```diff
@@ -0,0 +1,324 @@
+
+.. _rest_system:
+
+v6 System Services
+===============
+
+These services provide access to general system information.
+
+.. _rest_v6_system_status:
```

Comment:
Add in the sub-headings (use --- instead of ===) for each service like the v6 batch services.